main.go

```diff
@@ -88,11 +91,16 @@ func newApp(ctx context.Context, port, projectID string) (*App, error) {
 	}
 	app.projectID = projectID
 
-	l, err := jsonlog.NewLogger(fmt.Sprintf("projects/%s", app.projectID))
+	client, err := logging.NewClient(ctx, fmt.Sprintf("projects/%s", app.projectID),
+		// We don't need to make any requests when logging to stderr.
+		option.WithoutAuthentication(),
```

Comment:
Do we still need these options when we're not making API calls?

Reply:
These options ensure the libraries don't try to make to lookup credentials with ADC. This is done incase the env does not have creds to provide -- as they are not needed when using the logging package in this manner. The second option is something that we should eventually do away with and may be able to be implicitly set by the first in the future.